src/client/Client.java

```diff
@@ -195,13 +195,13 @@ private void playAudio(String audioPath, boolean isBgm) {
             if (isBgm) {
                 currBgm = audioPath;
                 bgmStream = AudioSystem.getAudioInputStream(audioFile);
-                bgmClip = AudioSystem.getClip();
+                bgmClip = AudioSystem.getClip(null);
                 bgmClip.open(bgmStream);
                 bgmClip.start();
                 bgmClip.loop(Clip.LOOP_CONTINUOUSLY);
             } else {
                 soundStream = AudioSystem.getAudioInputStream(audioFile);
-                soundClip = AudioSystem.getClip();
+                soundClip = AudioSystem.getClip(null);
```

Comment:
Why is null required to be an argument?

Reply:
i literally have no idea why
https://stackoverflow.com/a/36101885
something about ubuntu i think

Reply:
If this is required for the app to work across platforms, that's fine